docker/ssh_data/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && \
 # add a user 'geoserver' with default password : 'geoserver'
 # that can be overriden by environement variable $USER_PASS
 RUN groupadd --gid 999 geoserver
-RUN useradd -ms /bin/bash --home /home/geoserver -p wtw.aC5paV.qU --uid 999 --gid 999 geoserver
+RUN useradd -ms /bin/bash --home /home/geoserver -p $(perl -e'print crypt("${USER_PASS:-geoserver}", "salt")') --uid 999 --gid 999 geoserver
```

Comment:
you are right ! and you will set the new password when you will run the image : 
docker run -d --name georchestra_geodata_1 --env USER_PASS=another_password georchestra/ssh_data